### PR TITLE
DF-425 add adjourned event, handle 'deleted' events in elasticsearch

### PR DESCRIPTION
**Pull request by fabiangotzen**

### CODE REVIEW

File: infrastructure/src/main/java/ch/admin/seco/jobs/services/jobdesk/infrastructure/messagebroker/JobAdvertisementEventSink.java [hunk 1]
```diff
@@ -37,17 +37,17 @@ public JobAdvertisementEventSink(JobAdvertisementRepository jobAdvertisementRepo
...
                 .ifPresent(jobdeskJobApplicationService::save);
         }
-        if (isOfEventtypeArchivedOrCancelled(eventType)) {
+        if (hasToDelete(eventType)) {
             this.jobdeskJobApplicationService.deleteJobById(jobAdvertisementEvent.getId());
         }
     }
 
-    private boolean isOfEventypePublishedPublicOrUpdated(JobAdvertisementEvent eventType) {
+    private boolean hasToSave(JobAdvertisementEvent eventType) {
```
[the remark this answers is not in the document]
Comment (varusakov) [reply infrastructure/src/main/java/ch/admin/seco/jobs/services/jobdesk/infrastructure/messagebroker/JobAdvertisementEventSink.java]: As proposal - `isSaveEvent` and `isDeleteEvent`

File: core/src/main/java/ch/admin/seco/jobs/services/jobdesk/core/domain/events/DomainEventType.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
+package ch.admin.seco.jobs.services.jobdesk.core.domain.events;
+
+import java.util.Objects;
+
+import javax.persistence.Embeddable;
+
+@Embeddable
```
Comment (varusakov) [inline core/src/main/java/ch/admin/seco/jobs/services/jobdesk/core/domain/events/DomainEventType.java]: Do we save this events to DB?

File: infrastructure/src/main/java/ch/admin/seco/jobs/services/jobdesk/infrastructure/messagebroker/JobAdvertisementEventSink.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (fabiangotzen) [reply infrastructure/src/main/java/ch/admin/seco/jobs/services/jobdesk/infrastructure/messagebroker/JobAdvertisementEventSink.java]: Changed

File: core/src/main/java/ch/admin/seco/jobs/services/jobdesk/core/domain/events/DomainEventType.java (hunk 2 above)
Comment (fabiangotzen) [reply core/src/main/java/ch/admin/seco/jobs/services/jobdesk/core/domain/events/DomainEventType.java]: No, I don't think thats necessary. Removed the annotation and the Constructor.

**Pull request merged**